features/page_objects/journey/location_page.rb

```diff
@@ -1,13 +1,15 @@
 class LocationPage < SitePrism::Page
 
+  # This page object is currently shared between old and new apps, as the elements are similar.
+  # Update once the old registration journey is no longer live.
+
   # Where is your principal place of business?
-  element(:heading, "#groupLabel")
```

Comment:
Why removing this?

Reply:
There are two heading elements in this file and the one above is only valid for the old app. The remaining one applies to both the old and new app :)